packages/browser/src/index.ts

```diff
@@ -86,6 +86,47 @@ export default class LogtoClient {
     sessionStorage.setItem(this.logtoSignInSessionKey, jsonItem);
   }
 
+  public isAuthenticated() {
+    return Boolean(this.idToken);
+  }
+
+  public getAccessToken(resource = '', scopes = []): Optional<string> {
```

Comment:
per discussed today `scopes` should be removed

Reply:
also we allow empty / null but no default value for these params

Reply:
Thanks, code updated